bgpd/bgp_route.c

```diff
@@ -1503,9 +1493,14 @@ int bgp_evpn_path_info_cmp(struct bgp *bgp, struct bgp_path_info *new,
 {
 	enum bgp_path_selection_reason reason;
 	char pfx_buf[PREFIX2STR_BUFFER] = {};
+	bool debug = false;
 
-	return bgp_path_info_cmp(bgp, new, exist, paths_eq, NULL, 0, pfx_buf,
-				AFI_L2VPN, SAFI_EVPN, &reason);
+	if (debug)
```

Comment:
Hm, can this somehow be true? :shrug:

Reply:
no it cannot but it's setup to do the right thing whenever someone adds the ability to debug evpn bestpath via `debug bgp bestpath XXX`.  This PR just fixes the code pattern that was using bgp_pathinfo_cmp wrong.